keps/2724-topology-aware-scheduling/README.md

```diff
@@ -430,6 +430,11 @@ const (
   // at the highest topology level, then it gets admitted as distributed
   // among multiple topology domains.
   PodSetPreferredTopologyAnnotation = "kueue.x-k8s.io/podset-preferred-topology"
+
+  // PodSetUnconstrainedTopologyAnnotation indicates that a PodSet requires does not have any topology requirements.
+  // Kueue admits the PodSet if there's enough free capacity anywhere in the cluster.
```

Comment:
```suggestion
  // Kueue admits the PodSet if there's enough free capacity available.
```
not necessarily in the whole cluster. This is still constrained by the RF.spec.nodeLabels.